rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "1.76"
+profile = "default"
+components = ["rust-analyzer"]
```

Comment:
Is it possible to make "nightly" the default toolchain for `rustfmt`/`cargo fmt`?

Reply:
Looking at https://github.com/rust-lang/rustup/issues/3546#issuecomment-1931848506 again (I already linked to it in https://github.com/martinvonz/jj/pull/4613#issuecomment-2405901050), probably not :(

Reply:
One thing we could do is add editor configuration to the repo which overrides the formatting command. That would be a bunch more files though and I assume some people won't like it.

Reply:
I'm not personally opposed to adding per-editor configuration to the repo, but I'll leave it as a potential follow-up.